### DISCUSSION TITLE
[Coreshop Datahub] Assigning the coreshop_coreshop_relation to an object

### DISCUSSION THREAD

**Discussion by twin-elements** [pimcore/pimcore#16906] [Q&A / Support] [answered]
Using the DataHub importer (Load DataObject transformation), I load a CoreShopManufacturer object and need to assign it to a product.

![image](https://github.com/coreshop/CoreShop/assets/17727266/f2959f83-b0c7-4cff-9dde-22f29403cb42)

But in the data Target it is not possible to assign a manufacturer (not available).

![image](https://github.com/coreshop/CoreShop/assets/17727266/7390dd28-267f-4577-80d7-29be1ad89505)

Is it possible to assign a coreshop_coreshop_relation to an object?

**Comment by dpfaffenbauer** [comment, marked as answer]
Hey, you opened the same question in the CoreShop Forum and here. Usually one is enough, so follow up here please: https://github.com/coreshop/CoreShop/discussions/2597
